src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/SupplierStatementRepositoryTest.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.reform.blobrouter.data;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.blobrouter.data.reconciliation.SupplierStatementRepository;
+import uk.gov.hmcts.reform.blobrouter.data.reconciliation.model.EnvelopeSupplierStatement;
+import uk.gov.hmcts.reform.blobrouter.data.reconciliation.model.NewEnvelopeSupplierStatement;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Optional;
+import java.util.UUID;
+import javax.validation.ClockProvider;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+@ActiveProfiles({"integration-test", "db-test"})
+@SpringBootTest
+public class SupplierStatementRepositoryTest {
+
+    @Autowired SupplierStatementRepository repo;
+    @Autowired ObjectMapper objectMapper;
+    @Autowired ClockProvider clockProvider;
+
+    @Test
+    void should_save_and_find_statement() throws Exception {
+        // given
+        var statement =
+            new NewEnvelopeSupplierStatement(
```

Comment:
no need for that new line right? also not important comment

Reply:
imho more readable